src/apiCalls.tsx

```diff
@@ -0,0 +1,48 @@
+/*-----------------------------------// GET //--------------------------------------*/
+
+export const getUser = async (userId: number) => {
+  console.log(userId, '---> HIT GET USER')
+  try {
+    const response = await fetch(`http://localhost:3001/api/v1/users/${userId}`, {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+
+    if (!response.ok) {
```

Comment:
Good looking out to gracefully guide the user with the response.status and also having developer empathy by throwing a console.log in for good measure.

Reply:
Yeah! All good brotha!